src/transforms/group.js

```diff
@@ -319,6 +320,15 @@ function reduceAccessor(f) {
   };
 }
 
+function reduceMaybeTemporalAccessor(f) {
+  return {
+    reduce(I, X) {
+      const x = f(I, (i) => X[i]);
+      return isTemporal(X) ? new Date(x) : x;
+    }
+  };
+}
```

Comment:
small optimization:
```suggestion
function reduceMaybeTemporalAccessor(f) {
  return {
    scope: "data",
    reduce(I, X, context) {
      if (arguments.length === 2) return isTemporal(X);
      const x = f(I, (i) => X[i]);
      return context ? new Date(x) : x;
    }
  };
}
```

if we do this, we'll also have to initialize the context in https://github.com/observablehq/plot/blob/21dc1c70f4de1ec2a63ae49fce6f04089bed3d5f/src/channel.js#L66

Reply:
I’d prefer to keep this simple for now.